Dockerfile

```diff
@@ -0,0 +1,33 @@
+# vim:set ft=dockerfile:
+ARG BASE_IMAGE_TAG
+FROM pavics/workflow-tests:${BASE_IMAGE_TAG:-latest}
+ARG DEBIAN_FRONTEND=noninteractive
+ENV PIP_ROOT_USER_ACTION=ignore
+LABEL org.opencontainers.image.authors="https://github.com/ouranosinc/pavics-sdi"
+LABEL Description="PAVICS-SDI-TESTING-IMAGE" Vendor="Birdhouse" Version="1.3.0"
+
+# root-level commands
+USER root
+
+# Install build tools
+RUN apt-get install --yes build-essential \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+# Switch to /code directory
+WORKDIR /code
+
+# Build finch environment
+COPY environment-dev.yml /code
+RUN mamba install -n birdy python=3.8 --yes \
```

Comment:
What?  The Jupyter image is the exact same one used by Jenkins, it should already have everything needed to run tests, why install again?  And it's supposed to be python=3.9 now.